version.txt

```diff
@@ -1 +1 @@
-0.9.0
```

Comment:
This MUST be reset back to just 0.9.0! Because it will cause many issues like this!